app/database_etl/test_adjustment_handler/test_adjustment.py

```diff
@@ -178,16 +209,20 @@ def get_adjusted_estimate(self, estimate: pd.Series) -> Tuple:
 
```

Comment:
We still have the "model sets" thing in `pystan_adjust`. Is running several model sets and ensuring consistency between them still necessary? Especially if we are testing if things are bounded? I suspect not, and if so, we will need to refactor / remove here

Reply:
Yes, I removed this part.